### PR TITLE
Updating the UI of Dark/Light mode button

### PR DESCRIPTION
**Pull request by Lobi29**
<h1 align="center">
Chaing the UI of Dark/Light Mode Button #268 
</h1>

### Screenshot

- dark mode
Screenshot from 2024-02-28 12-07-28

- light mode

Screenshot from 2024-02-28 12-07-19

### Feature
- Enhancing the user experience with seamless transitions for a smoother journey

### CODE REVIEW

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Comment by rithviknishad** [comment]
@Lobi29 deploy failing due to lint errors, could you run `pnpm lint-fix` along with the upcoming changes?

image

**Comment by Ashesh3** [comment]
For future, you can also configure [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) in your code editor and auto format the file as you are working on it.

**Comment by rithviknishad** [comment]
> For future, you can also configure [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) in your code editor and auto format the file as you are working on it.

@Lobi29 support for that has been added in https://github.com/coronasafe/leaderboard/commit/6e12824aba7ed5d1cfdd68f59028c909e4c2e672

Pulling the latest main to this branch should help

**Comment by Lobi29** [comment]
Thank your for suggestions,

File: components/ThemeSwitch.tsx [hunk 1]
```diff
@@ -1,47 +1,41 @@
...
+import { MdOutlineDarkMode } from "react-icons/md";
 
 export default function ThemeSwitch({ className }: { className?: string }) {
   const { theme, setTheme } = useTheme();
 
+  useState(() => {
+    setTheme("dark");
+  });
+  
+
```
[the remark this answers is not in the document]
Comment (Lobi29) [reply components/ThemeSwitch.tsx:17]: Cuz I want the default theme of the website as Dark Mode. If you want light mode as default theme I can update as per your requirement 👍

[the remark this answers is not in the document]
Comment (rithviknishad) [reply components/ThemeSwitch.tsx:17]: No, instead let's remove this setting of default from our side. 

Currently in https://contributors.ohc.network when you switch it to light mode, it'll stay in light mode upon re-opening. If you switch it to dark mode, it'll stay in dark mode upon re-opening. 

It'll use whatever the user has set last.

[the remark this answers is not in the document]
Comment (rithviknishad) [reply components/ThemeSwitch.tsx:17]: If some light mode fan tries to open the one this PR made, they'll always have to keep switching it to light mode manually right?

[the remark this answers is not in the document]
Comment (Lobi29) [reply components/ThemeSwitch.tsx:17]: Okay 👍, 😊

**Pull request closed without merging**